structures/PenguClient.js

```diff
@@ -1,14 +1,16 @@
 const { Client, PermissionLevels } = require("klasa");
+const MusicManager = require("../lib/music/MusicManager");
 
 class PenguClient extends Client {
 
     constructor(options) { //eslint-disable-line
         super(options);
         this.config = require("../config.json");
         this.functions = require("../utils/functions.js");
-        this.lavalink = null;
-        this.queue = new Map();
         this.log = require("../utils/log.js");
+
+        this.lavalink = require("../lib/music/LavalinkClient");
+        this.music = new MusicManager(this);
```

Comment:
Not needed

Reply:
Non of these?

Reply:
the review is for the line not the block, only music